openapi/tests/openapi_integration/test_ping_api.py

```diff
@@ -1,14 +1,14 @@
 from .helpers.helpers import request_with_validation
 
-def test_health_api():
+def test_ping_api():
```

Comment:
Nitpick: maybe just `test_root` or `test_root_api`.

Reply:
Good one. I was confused what to name it. I'll update the OpenAPI docs as well.